src/crlf.c

```diff
@@ -225,24 +246,49 @@ static int crlf_apply_to_workdir(
 	struct crlf_attrs *ca, git_buf *to, const git_buf *from)
 {
 	const char *workdir_ending = NULL;
+	git_buf_text_stats stats;
+	int isBinary;
```

Comment:
again, please use something like `is_binary` here, or just `binary`.  and it looks like it should be a `bool`, not an `int`.